include/flexflow/ops/concat.h

```diff
@@ -10,12 +10,20 @@
 
 namespace FlexFlow {
 
+struct ConcatInputShape {
```

Comment:
Okay I think I was wrong on this, let's just make `std::vector<T>` hashable in `hash_utils.h`